src/components/InventoryGroups/InventoryGroups.js

```diff
@@ -28,6 +27,11 @@ const InventoryGroups = () => {
         }
 
         setIsLoading(false);
+    };
+
+    useEffect(async () => {
```

Comment:
Maybe we can improve this further. Like in the react docs: https://react.dev/learn/synchronizing-with-effects#fetching-data

Reply:
@bastilian, oh, nice, thanks for sharing. Anyway, for which scenario this would be useful here? The fetch function, ones finished, updates only the component state variables and it doesn't do anything global (in terms of redux store handling, for instance).

Reply:
Here for example, it could be used to prevent it calling the setState functions in `checkForGroups`, as this could result in it being called in for some reason this component was unmounted, which would throw an error.